pdchaosazure/machine/actions.py

```diff
@@ -136,6 +136,7 @@ def restart_machines(filter: str = None,
 
 def stress_cpu(filter: str = None,
                duration: int = 120,
+               number_of_cores: int = 0,
```

Comment:
Let's get rid of cores as we cannot address particular cores or number of cores that we want to stress

Reply:
Done